diff --git a/setup.py b/setup.py
index 400efc7..d3bc4b0 100644
--- a/setup.py
+++ b/setup.py
@@ -12,8 +12,8 @@ setup(
     url = 'https://github.com/blakev/python-syncthing',
     license = 'The MIT License',
     install_requires = [
-        'python-dateutil>=2.8.1,<=2.8.2',
-        'requests>=2.24.0,<=2.28.0'
+        'python-dateutil>=2.8.1',
+        'requests>=2.24.0'
     ],
     extras_require = {
         'dev': [
